Resources/views/CRUD/base_list.html.twig

```diff
@@ -39,6 +39,7 @@ file that was distributed with this source code.
                             {% block table_header %}
                                 <thead>
                                     <tr class="sonata-ba-list-field-header">
+                                        {% block table_header_row_start %}{% endblock table_header_row_start %}
```

Comment:
@pulzarraider it might indeed, but in that case we would need to rename most of the twig blocks to stay coherent. This would introduce a lot of BC Breaks though. Coucou!!! @rande